game/src/test/scala/io/aigar/game/PlayerSpec.scala

```diff
@@ -46,7 +46,7 @@ class PlayerSpec extends FlatSpec with Matchers {
 
     player.onExternalAction
 
-    player.isActive() shouldBe true
+    player shouldBe 'active
```

Comment:
[this](http://www.scalatest.org/user_guide/using_matchers#checkingBooleanPropertiesWithBe)